src/App.tsx

```diff
@@ -1,8 +1,66 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { getTodos } from './api/api';
+import { TodoList } from './Todolist';
+import { Buttons } from './Buttons';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+const App = () => {
+  const [todosList, setTodos] = useState<Todo[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isVisible, setIsVisible] = useState<boolean>(false);
+
+  const loadTodos = () => {
+    setIsLoading(!isLoading);
+    setIsVisible(!isVisible);
+    setTimeout(() => {
+      getTodos()
+        .then((data) => setTodos(data))
+        .finally(() => {
+          setIsLoading(false);
+        });
+    },
+    1000);
+  };
+
+  const sortByFilter = (filter: string) => {
```

Comment:
```suggestion
  const sortByColumn = (filter: string) => {
```
Сортировка по фильтру выглядит странно

Reply:
```suggestion
  const sortByField = (filter: string) => {
```
можно еще так